scripts/manage-db.py

```diff
@@ -13,22 +14,74 @@
 
 from auslib.db import AUSDatabase
 
+def cleanup_releases(trans, nightly_age, dryrun=True):
+    # This and the subsequent queries use "%%%%%" because we end up going
+    # through two levels of Python string formatting. The first is here,
+    # and the second happens at a low level of SQLAlchemy when the transaction
+    # is being executed.
+    query = """
+LEFT OUTER JOIN releases_history USING (name)
+LEFT JOIN rules ON (name=mapping)
```

Comment:
I'll preface all these comments by saying my SQL is a smidgen rusty so I'm not sure I 100% understand how this works, but tally ho chaps.

Is there any difference between LEFT JOIN and LEFT OUTER JOIN ? The docs indicate OUTER is optional in MySQL. The queries are a bit intensive to play around with on (r/o) prod, and there's not enough history in docker, so I couldn't experiment.

Speaking of expensive, I think the query need only consider the most recent row in releases_history for each release (the old procedure deletes from releases and releases_history, but you're only acting on releases here). I wonder if there's a condition we can use to exclude processing any older releases_history rows, and leave cleanup_releases_history() to handle those.